docker/localsettings.py

```diff
@@ -94,6 +94,21 @@
     }
 
     WAREHOUSE_DATABASE_ALIAS = 'warehouse'
+    
+USE_CITUSDB = os.environ.get('USE_CITUSDB', 'no') == 'yes'
+if USE_CITUSDB:
+    DATABASES['citus'] = {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'DISABLE_SERVER_SIDE_CURSORS': True,
+        'NAME': 'commcarehq_citus',
```

Comment:
Is this citus cluster expected to be the only database using citus in HQ forever, or is this one just for UCR (or something else) and we may possibly add others later? Would it make sense to use a name that indicates what it is being used for?

Reply:
good point, I'll change it